docs/README.md

```diff
@@ -68,14 +68,68 @@ macOS (OS X) development environment.
 # User documentation
 
 * [Getting started (How-To with examples)](usage.md)
+
+## C/C++ Analysis
 * [Analyzer User guide](analyzer/user_guide.md)
-* [Webserver User guide](web/user_guide.md)
-* [Requiring credentials to view analysis results](web/authentication.md)
-* [Overview about connecting multiple analysis run databases](web/products.md)
-* [Permission management](web/permissions.md)
-* [Usage of PostgreSQL database](web/postgresql_setup.md)
-* [How to deal with false positives](analyzer/false_positives.md)
-* [Store Cppcheck analyzer reports](/tools/report-converter/README.md#cppcheck)
+* [Avoiding or suppressing false positives](analyzer/false_positives.md)
+* [Checker and Static Analyzer configuration](analyzer/checker_and_analyzer_configuration.md)
+* [GCC incompatibilities](analyzer/gcc_incompatibilities.md)
+* [Suppressing false positives](analyzer/user_guide.md#source-code-comments)
+
+## Web based report management
+* [Webserver User Guide](web/user_guide.md)
+* [Command line and WEB UI Feature overview](feature_comparison.md)
+* Security configuration 
+** [Configuring Authentication](web/authentication.md)
```

Comment:
You have to use one start (`*`) and indent the line with 2 spaces to get a sublist.

Reply:
done as you suggested